apps/web/tailwind.config.js

```diff
@@ -9,8 +9,11 @@ const {VeniceTheme} = require('./styles/themes')
  */
 module.exports = {
   content: [
-    './{components,pages,screens}/**/*.{ts,tsx}',
-    '../../integrations/*/**/*.tsx',
+    './components/**/*.tsx',
+    './layouts/**/*.tsx',
```

Comment:
Forgot to add this in the previous PR; that was why I had an issue with tailwind (it didn't generate new classes, so only utility classes used else where worked).

Reply:
Ah i see. makes sense.